elodie/tests/geolocation_test.py

```diff
@@ -95,30 +96,33 @@ def test_reverse_lookup_with_invalid_key():
 def test_lookup_with_valid_key():
     res = geolocation.lookup(location='Sunnyvale, CA')
     latLng = res['results'][0]['locations'][0]['latLng']
-    assert latLng['lat'] == 37.36883, latLng
-    assert latLng['lng'] == -122.03635, latLng
+    assert latLng['lat'] == 37.37187, latLng
```

Comment:
Float equality is **very** fragile, for a variety of reasons. If updating this test I would recommend using some sort of delta/fuzz factor. Based on the current style you might like the approach mentioned in:

* https://randycoulman.com/blog/2018/06/19/comparing-floats-in-tests/ - uses a class to override equality compare and your existing code structure would remain

Alternatives:

* Python 3.5>= https://davidamos.dev/the-right-way-to-compare-floats-in-python/
* https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertAlmostEqual more version compat
* you could always roll your own :-)